doc/fuzzing.md

```diff
@@ -0,0 +1,59 @@
+# Fuzzing
+
+## Setting up a fuzzer locally
+
+As of v1.8, OpenConsole can now be built with a `Fuzzing` configuration.
+
+To set up a fuzzer, you'll need an `LLVMFuzzerTestOneInput` function. This serves as a way for the fuzzer to attach itself and inject tests into your fuzz target.
+
+To build the fuzzer locally, build the OpenConsole solution in the `Fuzzing` configuration. This should output an executable that runs the fuzzer on the provided test case. In the case of PR \#9604, the desired executable is located at `bin\x64\Fuzzing\OpenConsoleFuzzer.exe`.
+
+**NOTE**: If you get an error like `cannot open input file libsancov.lib and clang-rt.fuzzer-x86_64.lib`, install the "C++ AddressSanitizer" via the Visual Studio Installer.

```

Comment:
Is this something we should add to the `.vsconfig` file to ensure it gets installed?